.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -0,0 +1,4 @@
+cleanup_settings_version=2
+eclipse.preferences.version=1
```

Comment:
Will look up on how to put this file using git ignore! Thanks for the guidance!(: